action.yaml

```diff
@@ -0,0 +1,117 @@
+name: Install MDAnalysis
+description: Install MDAnalysis
+inputs:
+  version:
+    description: >
+      Version of MDAnalysis to install.
+      Can be a specified version (e.g. 2.0.0),
+      "develop", or "latest".
+    required: true
+    default: latest
+  install-tests:
+    description: Whether to install MDAnalysisTests
+    required: true
+    default: false
+  installer:
+    description: >
+      Installer to use.
+      Must be one of pip, conda, or mamba.
+    required: true
+    default: pip
+  shell:
+    description: >
+      Which shell to use.
+      If you have a conda environment, you should
```

Comment:
It came to mind that I've never checked if plain bash would just work for everything, I assume you have?

Reply:
Yup. It doesn't. You need a login shell to source conda